popde/adaptive_kde.py

```diff
@@ -291,6 +296,21 @@ def loo_cv_score(self, rescale_factors_alpha):
             float: Negative sum of log likelihood over test data.
         """
         rescale_val = rescale_factors_alpha[:-1]
+        # Ensure rescale_val is a numpy array
+        rescale_val = np.array(rescale_val)
```

Comment:
can this not be combined with the previous line?  Also the comment is unnecessary, it is obvious that using the `np.array()` function ensures is a numpy array